plugins/org.python.pydev/pysrc/runfiles.py

```diff
@@ -72,10 +72,13 @@ def main():
         else:
             raise ImportError()
 
-    except ImportError:
+    except ImportError as e:
```

Comment:
I don't think the 'as e' syntax is valid for older Python versions, so, it can't be used (PyDev must support Python 2.4 onwards).